DB/src/main/java/io/deephaven/db/tables/ColumnDefinition.java

```diff
@@ -391,11 +397,17 @@ void setComponentType(Class componentType) {
         this.componentType=componentType;
     }
 
-    private int columnType=Integer.MIN_VALUE;
+    private int columnType = Integer.MIN_VALUE;
     public int getColumnType() {
         return columnType;
     }
 
+    private boolean hasSymbolTable = false;
+    public boolean hasSymbolTable() { return hasSymbolTable; }
```

Comment:
You need to delete `getSymbolTableType()` (and its enum) and replace it with this, or change that method to do:
```
    public SymbolTableType getSymbolTableType() {
        if (!CharSequence.class.isAssignableFrom(dataType) || !isDirect()) {
            return null;
        }
        if (hasSymbolTable) {
            return SymbolTableType.COLUMN_LOCATION;
        }
        return SymbolTableType.NONE;
    }
```